strongbox-web-core/src/main/java/org/carlspring/strongbox/converters/RoleListFormToRoleListConverter.java

```diff
@@ -16,17 +16,19 @@
         implements Converter<RoleListForm, List<RoleDto>>
 {
 
+    private RoleFormToRoleConverter roleFormToRoleConverter = new RoleFormToRoleConverter();
```

Comment:
we could use singleton instance like `AccessModelFormToUserAccessModelDtoConverter.INSTANCE`

Reply:
fixed